views/forms/ParticipantsSearchForm/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react'
+import { useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { InputAdornment } from '@mui/material'
+import SearchIcon from '@mui/icons-material/Search'
+import * as yup from 'yup'
+
+import ControlledMultiSelect from '../ControlledMultiSelect'
+
+const schema = yup.object({
+  participants: yup.array(),
+})
+
+const ParticipantsSearchForm = ({ initialValues, onSubmit, allOptions }) => {
+  const { handleSubmit, control, formState, watch } = useForm({
+    defaultValues: initialValues,
+    resolver: yupResolver(schema),
```

Comment:
should we use a mode: 'onChange' for validations or only when submitting..?

Reply:
good idea - https://github.com/AMP-SCZ/dpdash/pull/479/commits/b898e7a9465094559e5e9939062fe5810a2ba309